css/index.css

```diff
@@ -104,14 +143,44 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Nice work allowing the parent style the boxes except for the color which is overriden.